### ISSUE TITLE
prettier-plugin-svelte

### ISSUE THREAD

**Issue by basaran** [dprint/dprint-plugin-prettier#7]
Hello,

I'm trying to get dprint to work with prettier using the `prettier-plugin-svelte`. Is this supported?

I confirmed I can format fine with the local prettier installed in the project, but dprint says no files were found with the given plugins.

```
➜  something  dprint fmt src/routes/index.svelte
No files found to format with the specified plugins. You may want to try using `dprint output-file-paths` to see which files it's finding.
```

**Comment by dsherret** [comment]
@basaran sorry for the delay. This is implemented now in 0.3.0 (see [releases](https://github.com/dprint/dprint-plugin-prettier/releases))

**Comment by basaran** [comment]
Thank you, much appreciated. I tested that it works fine and well.
